synapse/models/risk.py

```diff
@@ -648,12 +648,21 @@ def getModelDefs(self):
                 )),
 
                 ('risk:vulnerable', {}, (
+
                     ('vuln', ('risk:vuln', {}), {
                         'doc': 'The vulnerability that the node is susceptible to.'}),
+
                     ('period', ('ival', {}), {
                         'doc': 'The time window where the node was vulnerable.'}),
+
                     ('node', ('ndef', {}), {
                         'doc': 'The node which is vulnerable.'}),
+
+                    ('mitigated', ('bool', {}), {
+                        'doc': 'Set to true if the vulnerable node has been mitigated.'}),
+
+                    ('mitigation', ('risk:mitigation', {}), {
```

Comment:
Does this need to be an array? What if there are two or three steps you took to mitigate the vuln?